asm/instruction_test.go

```diff
@@ -176,12 +176,19 @@ func TestInstructionsRewriteMapPtr(t *testing.T) {
 	}
 }
 
+type Comment string
```

Comment:
@lmb What are your thoughts on putting this in the package proper? I think providing this as part of the DSL makes sense, I see little point in letting the caller declare this manually.

Reply:
I don't feel strongly about it, but would probably not include it in the package. It's four lines of code.

Reply:
It makes sense to me to include it in the package, I know it is only 4 lines, but it saves the user having to figure this out. So I have to agree with @ti-mo on this one.